core/templates/dev/head/profile/Preferences.js

```diff
@@ -35,6 +36,46 @@ oppia.controller('Preferences', ['$scope', '$http', '$rootScope', '$modal', '$ti
     _saveDataItem('user_bio', userBio);
   };
 
+  $scope.changedAtLeastOnce = false;
+  $scope.subjectInterestsWarningText = null;
+  $scope.TAG_REGEX = '^[a-z ]+$';
+
+  $scope.updateSubjectInterestsWarning = function(subjectInterests) {
+    var alpha = /^[A-Za-z]+$/;
```

Comment:
I do not know why, but it simply does not work if I declare anything different than what I originally had. 
Here's my new code:
$scope.newChoiceValidator = new RegExp('^[a-z ]+$');
new-choice-regex="<[newChoiceValidator]>" width="100%"

Now, the REGEX does not allow alphabetic characters.